mysql/datadog_checks/mysql/mysql.py

```diff
@@ -656,7 +663,7 @@ def _query_size_per_schema(self, db):
 
                 schema_size = {}
                 for row in cursor.fetchall():
-                    schema_name = str(row[0])
+                    schema_name = to_native_string(row[0])
```

Comment:
I think it might worth adding a integration test with a DB with `utf8_bin` for example.